onnx/defs/tensor/defs.cc

```diff
@@ -806,68 +810,106 @@ Example 2:
           [4.5, 5.9],
       ],
   ]
+Example 3:
```

Comment:
Is this for the new mode? Shouldn't it have "elem_index" attribute?

Reply:
yes I will update this